hummingbird/ml/operator_converters/onnx/onnx_operator.py

```diff
@@ -256,7 +262,10 @@ def convert_onnx_add(operator, device=None, extra_config={}):
     assert operator is not None
 
     initializers = extra_config[constants.ONNX_INITIALIZERS]
-    val = list(initializers[operator.raw_operator.origin.input[1]].float_data)
+    if operator.raw_operator.origin.input[1] not in initializers:
```

Comment:
Can we do like:
`val = None`
`if operator.raw_operator.origin.input[1] in initializers:`
bla
(basically avoid the if\else and use `val = None` as default case)

Reply:
good idea!